doc/v2/howto/index_en.rst

```diff
@@ -1,11 +1,37 @@
 HOW TO
-=======
+========
+
+PaddlePaddle allows users to flexibly set various command line parameters to control the model training or forecasting process. How to use please refer to:
```

Comment:
PaddlePaddle provides the users the ability to flexibly set various command line parameters to control the model training and inference process. Please refer to the following instructions on using PaddlePaddle